go/main/Dockerfile.tmpl

```diff
@@ -77,6 +77,11 @@ RUN curl -sSLO https://storage.googleapis.com/obs-ci-cache/beats/WpdPack_4_1_2.z
   --output-lib /libpcap/win/WpdPack/Lib/x64/libwpcap.a \
   --input-def /libpcap/win/WpdPack/wpcap.def
 
+RUN go install github.com/ibmdb/go_ibm_db/installer@v0.4.5 \
```

Comment:
- As the driver library version is hardcoded here. Is there a way we could remove the dependency and use the latest version?
- I don't think it would be possible as we run the `setup` script below by locating the installation folder. Just to check if its possible.